src/screens/Dashboard/Footer/style.scss

```diff
@@ -0,0 +1,104 @@
+@import "../../../scss/variables";
+@import "../../../scss/_base";
+
+// * {
+//   border: 1px solid #f3f3f3;
+// }
+
+
+#footer {
```

Comment:
Avoid the use of an id here. Switch to a class. Same for all id's used throughout here. Only use id in situations where you need to override and so on.

Reply:
changed to classname